testData/cases/dielectric_refraction/dielectricRefraction.fdtd.json

```diff
@@ -0,0 +1,103 @@
+{
+    "format": "FDTD Input file",
+    "__comments": "Example input with sgbc material",
+
+    "general": {
+        "timeStep": 10e-11,
+        "numberOfSteps": 2000
+    },
+    
+    "boundary": {
+        "xLower": {"type": "pec"},
+        "xUpper": {"type": "pec"},
+        "yLower": {"type": "pmc"},
+        "yUpper": {"type": "pmc"},
+        "zLower": {"type": "mur"},
+        "zUpper": {"type": "mur"}
+    },
+
+    "mesh": {
+        "grid": {
+            "numberOfCells": [2, 2, 200],
+            "steps": { "x": [0.025], "y": [0.025], "z": [0.025] }
+        },
+        "coordinates": [
+            {"id": 1, "relativePosition": [1, 1, 40]},
+            {"id": 2, "relativePosition": [1, 1, 160]},
+            {"id": 3, "relativePosition": [1, 1, 5]}
+        ],
+        "elements": [
+            {"id": 1, "type": "node", "coordinateIds": [1]},
+            {"id": 2, "type": "node", "coordinateIds": [2]},
+            {"id": 3, "type": "node", "coordinateIds": [3]},
+            {"id": 4, "type": "cell", "intervals": [ [ [-1, -1, 0], [3,3,10] ] ]},
+            {"id": 5, "type": "cell", "intervals": [[ [0, 0, 100], [2,2,200] ]]}
+        ]
+    },
+
+    "materials": [
+        {
+            "id": 1,
+            "name": "DielectricMaterial",
+            "type": "isotropic",
+            "relativePermittivity": 4
+        }
+
+    ],
+    "sources": [
+        {
+            "type": "planewave",
+            "magnitudeFile": "gauss.exc",
+            "elementIds": [4],
+            "direction": {
+                "theta": 0.0,
+                "phi": 0.0
+            },
+            "polarization": {
+                "theta": 1.5708,
+                "phi": 0.0
+            }
+        }
+    ],
+
+    "materialAssociations": [
+        {
+            "type": "bulk",
```

Comment:
`type` is no longer required within `materialAssociations`. We realized that it was mostly a source of confusion.